backend.py

```diff
@@ -2,41 +2,62 @@
 from flaskext.mysql import MySQL
 from datetime import datetime
 import logic
+import json
+from collections import defaultdict
 app = Flask(__name__)
 
-app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
-app.config['MYSQL_USER'] = 'Osama'
-app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_HOST'] = 'Osamas-MacBook-Pro.local'
+# app.config['MYSQL_USER'] = 'Osama'
+# app.config['MYSQL_PASSWORD'] = 'CSCI5030SLU2021'
+# app.config['MYSQL_DATABASE_DB'] = 'wordsense'
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_DATABASE_USER'] = 'pnkls'
+app.config['MYSQL_DATABASE_PASSWORD'] = 'root'
 app.config['MYSQL_DATABASE_DB'] = 'wordsense'
 
 mysql = MySQL(app)
 conn = mysql.connect()
 cursor = conn.cursor()
 
+def GetIndexing():
+    f = open("indexing.txt", "r")
```

Comment:
Variable f name should be changed, for good coding. Also, I agree with the suggestion by Osama.